pilot/pkg/model/extensions.go

```diff
@@ -47,6 +47,8 @@ const (
 	WasmPolicyEnv = "ISTIO_META_WASM_IMAGE_PULL_POLICY"
 	// name of environment variable at Wasm VM, which will carry the resource version of WasmPlugin.
 	WasmResourceVersionEnv = "ISTIO_META_WASM_PLUGIN_RESOURCE_VERSION"
+	// name of fallback policy for the download failure case
+	WasmDownloadFallbackPolicyEnv = "ISTIO_META_WASM_DOWNLOAD_FALLBACK_POLICY"
```

Comment:
should this be a part of API?

Reply:
Put it the other way, why is this not default?

Reply:
I am thinking about exposing the option in API surface later.

Reply:
@kyessenov basically, "allow all the traffic" is default.